docs/config.md

```diff
@@ -35,7 +35,7 @@ documented in `ipfs config profile --help`.
 - `local-discovery`
 
   Sets default values to fields affected by the server
-  profile, enables discovery in local networks.
+  profile enables discovery in local networks.
```

Comment:
Enables local discovery (enabled by default). Useful to re-enable local discovery after it's disabled by another profile (e.g., the `server` profile).